Package.swift

```diff
@@ -65,9 +65,8 @@ let package = Package(
                 "BrowserServicesKit"
             ],
             resources: [
-                .copy("UserScript/testUserScript.js"),
-                .copy("MockFiles"),
```

Comment:
`MockFiles` only contains `privacy-config-example.json` which is duplicated [here](https://github.com/duckduckgo/BrowserServicesKit/blob/03b42987d765e92b58a5975f6f6edad503100ec8/Tests/BrowserServicesKitTests/Resources/privacy-config-example.json). I compared these two files with a diff tool, they're the same so let's remove one. 🙂